examples/demo.css

```diff
@@ -42,7 +42,7 @@ h1 {
   margin: 0 !important;
 }
 
-.form-control {
+.form-control [type="range"] {
```

Comment:
There should be no space between `.form-control` and `[type="range"]`

Reply:
Thanks for the review. I've committed the changes. 

![space-removed](https://user-images.githubusercontent.com/37906201/51565213-fea8b200-1e56-11e9-9a56-3023e1600e44.png)

Reply:
@HarshKhandeparkar I assigned myself to provide 2nd approval after you approve, so we can request a merge promptly :)